pkgs/by-name/wa/wahay/package.nix

```
@@ -0,0 +1,60 @@
+{
+  lib,
+  stdenv,
+  buildGoModule,
+  fetchFromGitHub,
+  gnumake,
+  pkg-config,
+  gtk3,
+  git,
+  rubyPackages,
+}:
+
+buildGoModule rec {
+  pname = "wahay";
+  version = "0.2";
+
+  src = fetchFromGitHub {
+    owner = "digitalautonomy";
+    repo = "wahay";
+    rev = "v${version}";
+    hash = "sha256-59gVGDElIs+OQ/ELwOFjz6YqG18iHqoa2yYF2Ddi0/o=";
+  };
+
+  vendorHash = "sha256-w8lUnvy5dPMHWbzyyTq9Q/kE/4vSuOHffaY9CeasvQ0=";
+
+  buildInputs = [
+    gtk3
+  ];
+
+  nativeBuildInputs = [
+    pkg-config
+    gnumake
+    git
+    rubyPackages.sass
+  ];
+
+  #
+  buildPhase = ''
+    runHook preBuild
+
+    make BUILD_DIR=$out/bin TAG_VERSION=v${version} build
+
+    runHook postBuild
+  '';
+
+  checkFlags = [
+    "-skip=ConfigSuite.Test_CreateTempDir_createsTempFileInWahayDirectory"
+  ];
+
+  doCheck = false;
+  doInstallCheck = true;
+
+  meta = src.meta // {
+    description = "Easy-to-use, secure and decentralized conference call application ";
```

I’d recommend to avoid subjective language in the description and focus on factual information:

```suggestion
    description = "Decentralized encrypted conference call application";
```

I personally think that it would however be adequate to mention that it is _intended_ to be easy to use or to provide secure communication, which may be useful information. The communication being encrypted or more generally cryptographically protected is easily verifiable, whereas it actually providing security in a certain context is a more complicated matter.